google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/TableResultTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
Git thinks this is a new file. Should probably use 2017/2018 header with Google LLC from https://opensource.google.com/docs/releasing/preparing/#include-license-file-and-source-code-headers

Reply:
Changed to 2018, but same comment about "all rights reserved" as above.